### DISCUSSION TITLE
How to enable and configure the previewgenerator app within AIO docker container

### DISCUSSION THREAD

**Discussion by JartanFTW** [nextcloud/all-in-one#4184] [Questions] [answered]
In order for previewgenerator to function, it is necessary to setup a cronjob to run `occ preview:pre-generate` every so often (10 minutes standard). This can be done on the host machine as described in #2810, but this causes breakage if the docker containers are ever moved to another host. What is the proper procedure for setting up the previewgenerator cronjob within the NextCloud AIO container?

I did post [a comment regarding this](https://github.com/nextcloud/all-in-one/discussions/2810#discussioncomment-8374554) on in #2810 but it was marked as off-topic so I'm creating this discussion. I believe [issue #1 on the previewgenerator repository](https://github.com/nextcloud/previewgenerator/issues/1) is also relevant here.

**Comment by szaimen** [comment, marked as answer]
> How to enable and configure the previewgenerator app within AIO docker container

Short answer: you cannot. However you can via the host as you saw in: https://github.com/nextcloud/all-in-one/discussions/2810

The best solution would indeed be this: https://github.com/nextcloud/previewgenerator/issues/1. Feel free to upvote.
